_posts/2022-10-18-call-for-2023-volunteers.md

```diff
@@ -0,0 +1,300 @@
+---
+author: Drew Winstel
+category: General
+date: 2022-10-18 11:00:00-07:00
```

Comment:
I would make sure this date is newer than whatever time you publish it or it may not show up until the next time we publish something.

Reply:
Yep, planning on it.